MediaManager/Plugin.MediaManager.Android/MediaPlayerService/MediaPlayerService.cs

```diff
@@ -101,7 +101,11 @@ public override StartCommandResult OnStartCommand(Intent intent, StartCommandFla
         public override async Task Play(IMediaFile mediaFile = null)
         {
             await base.Play(mediaFile);
-            _mediaPlayer.PrepareAsync();
+            try
+            {
+                _mediaPlayer.PrepareAsync();
+            }
+            catch (Exception) { }
```

Comment:
Oh, yes, it should be IllegalStateException, not just Exception. I'm honestly not sure what to do when it is caught other than ignore it. Since this has been a very intermittent occurrence for me (I only know it's happening because of HockeyApp crash reports), I don't have any way of debugging it and inspecting the state. Looking online, the only reason that I could find for this exception is that it's already prepared. So I figured if it's already prepared, there is no need to prepare it again, so I just left the Catch block empty. If anyone has any other ideas of what to do here, I'm open.